### ISSUE TITLE
Support for stream processing variable length records

### ISSUE THREAD

**Issue by chandrasekaravr** [AbsaOSS/cobrix#189]
@yruslan 
I have large set of input files for processing. I wanted to take the stream processing approach. But, when i tried with the same options that i have successfully used for variable length records, like, 
`
.option("generate_record_id", true)
.option("is_record_sequence", "true")
.option("is_rdw_big_endian", "true")
.option("variable_size_occurs", "true")
`
It was getting processed as FixedLength records.

Any thoughts or suggestions?

**Comment by yruslan** [comment]
Yes, unfortunately streaming variable-record length files is not supported yet.
What is your use case? Do you want to stream from a Kafka topic?
